README.md

```diff
@@ -103,6 +104,105 @@ number of consenting nodes than PBFT.
 For more detailed description of the protocol, refer to [Efficient
 Byzantine Fault-Tolerance paper][minbft-paper].
 
+## Quick Start ##
+
+This quick start shows you how to runnning this project using Docker
+containers. This is the easiest way to try out this project with minimal
+setup. If you want to build and run without Docker, skip this section.
+
+### Prerequisites ###
+
+To run the containers, the following software must be installed on your
+system.
+
+ - [Docker Engine](https://docs.docker.com/engine/) (tested with version
+   19.03.6)
+ - [Docker Compose](https://docs.docker.com/compose/) (tested with
+   version 1.17.1)
+
+If you are using Ubuntu 18.04, they can be installed as follows:
+
+```sh
+sudo apt-get install docker.io docker-compose
+```
+
+Note that TEE is not required to run the containers; they will run in
+the simulation mode provided by Intel SGX SDK. We plan to have another
+container image which runs in the HW mode (i.e. using "real" hardware
+features) in a future release.
+
+### Building a Container Image ###
+
+Build an image of the containers as follows:
+
+```sh
+sudo docker build -f sample/docker/Dockerfile -t minbft .
+```
+
+Note that, by defaut, the `docker` command needs to be executed by
+`root`. Refer to [Docker's
+document](https://docs.docker.com/engine/install/linux-postinstall/#manage-docker-as-a-non-root-user)
+for details.
+
+If your system have the `make` command and the current user can run the
+`docker` command, the following command also can be used:
+
+```sh
+make docker
```

Comment:
Do we need `sudo` here, too?

Reply:
I will add `sudo` and delete "if ..., and the current user can run docker command"; I am using `sudo` in other `docker` command examples, so I think it will has consistency in the style by doing so.

Reply:
When we doing `sudo docker` it's more or less clear what could happen to the host system. On the other hand, `sudo make` could execute arbitrary command on the host system, and a careful user would have to check the Makefile...